src/axom/sidre/core/Group.hpp

```diff
@@ -1177,16 +1177,64 @@ class Group
              const Attribute* attr = nullptr) const;
 
   /*!
-   * \brief Load the Group from a file.
+   * \brief Load a Group hierarchy from a file into this Group
    *
-   * \param path      file path
-   * \param protocol  I/O protocol
+   * \param path     file path
+   * \param protocol I/O protocol
    * \param preserve_contents   Preserve existing contents of group if true
    */
   void load(const std::string& path,
             const std::string& protocol = SIDRE_DEFAULT_PROTOCOL,
             bool preserve_contents = false);
 
+  /*!
+   * \brief Load a Group hierarchy from a file into this Group, reporting
+   *        the Group name stored in the file
+   *
```

Comment:
This could use a brief explanation that the name of this Group isn't changed and that the calling code can optionally choose to use name_from_file, similar to what is said in for loadChild().

Reply:
I'll add this

Reply:
done